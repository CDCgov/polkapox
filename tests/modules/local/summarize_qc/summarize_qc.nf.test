nextflow_process {

    name "Test Process SUMMARIZE_QC"
    script "modules/local/summarize_qc/summarize_qc.nf"
    process "SUMMARIZE_QC"

    test("Summarize QC Test") {

        when {
            params {
                // define parameters here. Example:
                // outdir = "tests/results"
            }
            process {
                """
                // define inputs of the process here. Example:
                 input[0] = file("${projectDir}/tests/modules/local/summarize_qc/")
                 input[1] = file("${projectDir}/assets/samplesheet.test.csv")
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
