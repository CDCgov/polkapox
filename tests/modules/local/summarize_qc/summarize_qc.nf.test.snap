{
    "Summarize QC Test": {
        "content": [
            {
                "0": [
                    
                ],
                "1": [
                    
                ],
                "tsv": [
                    
                ],
                "versions": [
                    
                ]
            }
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "24.10.4"
        },
        "timestamp": "2025-07-11T16:26:32.613256056"
    }
}